Makefile

```diff
@@ -93,6 +93,10 @@ build/%.o: build/%.bin
 	@echo "  BIN   $@"
 	@$(OBJCOPY) -I binary -B aarch64 -O elf64-littleaarch64 $< $@
 
+build/font.o: font.b
+	@echo "  BIN   $@"
+	@$(OBJCOPY) -I binary -B aarch64 -O elf64-littleaarch64 $< $@
+
```

Comment:
Since this is exactly the same as the existing .bin rule call it font.bin and let it handle that automatically using the .bin rule. You can combine this with moving the font generation in as a makefile rule generating a .bin out of a ttf.